src/lib/utils/axis-utils.js

```diff
@@ -54,3 +44,16 @@ export function getTicksTotalFromSize(size) {
   }
   return 20;
 }
+
+/**
+ * Get the tick values from a given d3 scale.
+ * @param {d3.scale} scale Scale function.
+ * @param {number} tickTotal Total number of ticks
+ * @param {Array} tickValues Array of tick values if they exist.
+ * @returns {Array} Array of tick values.
+ */
+export function getTickValues(scale, tickTotal, tickValues) {
```

Comment:
can you add some tests for this? Now that it's pulled out as a util it seems like it'd be a lot easier to unit test it